docroot/themes/custom/uids_base/templates/layouts/onecol--background/layout--onecol--background.html.twig

```diff
@@ -21,16 +21,15 @@
 %}
 
 {% if background %}
-
   {% embed "@uids_base/uids/hero.html.twig" with {
     hero_alignment : 'bottom',
     'attributes' : attributes.addClass(classes)
   } %}
     {% block hero_image %}
-      {{ content.background }}
+      {{ content.background | without('layout_builder_add_block')}}
```

Comment:
I think these can be removed now. The buttons should not show up with the patch applied to lb_direct_add.